src/main/steps/index.ts

```diff
@@ -68,6 +68,15 @@ export const getNextIncompleteStepUrl = (req: AppRequest): string => {
   return `${url}${queryString}`;
 };
 
+export const getNextIncompleteConditionalOrderUrl = (req: AppRequest): string => {
```

Comment:
maybe this should be somewhere else, or more generic:

getNextIncompletePageAfter(req, page)